src/lib/containers/SearchV2.tsx

```diff
@@ -151,14 +140,27 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
       return false
     })
     if (indexOfColumn === -1) {
-      const columnSingleValueQueryFilter: ColumnSingleValueQueryFilter = {
```

Comment:
Used to always add a new ColumnSingleValueQueryFilter to the additionalFilters array.  Now, look at the columnModel that is associated to the selected columnName.  If the column type ends with '_LIST', then it's a multi-value column, so we should add a ColumnMultiValueFunctionQueryFilter instead.  Otherwise, do what it used to do (add a ColumnSingleValueQueryFilter)